### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by samuelory**

### CODE REVIEW

File: src/main/java/florizz/core/Parser.java [hunk 1]
```diff
@@ -21,13 +21,19 @@ public static Command parse (String input) throws FlorizzException{
         }
     }
 
-    private static AddBouquetCommand handleAddBouquet(String input){
-        String newBouquetName = input.substring(input.indexOf(" ") + 1);
+    private static AddBouquetCommand handleAddBouquet(String input) throws FlorizzException{
+        if (!input.contains(" ")){
```
Comment (Ijaaz01) [inline src/main/java/florizz/core/Parser.java:25]: Good job catching this bug

File: src/main/java/florizz/objects/Bouquet.java [hunk 2]
```diff
@@ -28,6 +29,6 @@ public boolean equals(Object obj) {
         Bouquet c = (Bouquet) obj;
 
         // Compare the data members and return accordingly
-        return Objects.equals(c.bouquetName, this.bouquetName);
+        return Objects.equals(c.bouquetName.toUpperCase(), this.bouquetName.toUpperCase());
```
Comment (Ijaaz01) [inline src/main/java/florizz/objects/Bouquet.java:32]: Good work on making the code more consistent

File: src/main/java/florizz/core/Parser.java [hunk 3]
```diff
@@ -21,13 +21,19 @@ public static Command parse (String input) throws FlorizzException{
         }
     }
 
-    private static AddBouquetCommand handleAddBouquet(String input){
-        String newBouquetName = input.substring(input.indexOf(" ") + 1);
+    private static AddBouquetCommand handleAddBouquet(String input) throws FlorizzException{
```
Comment (IanFH) [inline src/main/java/florizz/core/Parser.java:24]: Good job on adding the exception handling for blank space!

**Review by Ijaaz01 — APPROVED** [review]
Overall great work

**Pull request merged**